AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -650,6 +653,17 @@ - (void)scrollViewWillEndDragging:(UIScrollView *)scrollView withVelocity:(CGPoi
   }
 }
 
+- (void)scrollViewDidScroll:(UIScrollView *)scrollView{
+    for (ASCellNode *node in _cellsForVisibilityUpdates) {
+        if (node.shouldObserveVisibility) {
+            [node updateScrollSituationWithScrollVIew:scrollView];
+        }
+    }
+    if ([_asyncDelegate respondsToSelector:@selector(scrollViewDidScroll:)]) {
```

Comment:
Can you add a Boolean instance variable to cache the result of this? That would need to be set in the setAsyncDelegate method.